tasks/get_ca_certificate.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Check if ca certificate exists
+  stat: 
+    path: "{{ certificates_ca_key_dir }}/ca.crt"
+  register: certificate_ca_cert_path_stats
+  delegate_to: "{{ certificates_ca_server }}"
+
+- name: Generate new CA
+  include: generate_ca.yml
+  when: not certificate_ca_cert_path_stats.stat.exists
+  delegate_to: "{{ certificates_ca_server }}"
+
+- name: Slurp ca certificate
+  slurp:
+    src: "{{ certificates_ca_key_dir }}/ca.crt"
+  register: ca_certificate
+  delegate_to: "{{ certificates_ca_server }}"
+
+- name: Copy ca certificate to local host
+  copy:
+    content: "{{ ca_certificate['content'] | b64decode }}"
+    dest: "{{ certificates_local_ca_path }}"
```

Comment:
Don't you want to set some permissions on this?

Reply:
fixed